youtube-client/src/app/auth/pages/auth/auth.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import LoginService from 'src/app/auth/services/login.service';
+
+@Component({
+  selector: 'app-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.scss'],
+})
+export default class AuthComponent implements OnInit {
+  hide: boolean = true;
+
+  form: FormGroup;
+
+  buttonValue: string;
```

Comment:
rethink this part. You should show form when user is not logged and only log out button when user is logged-in